schema.json

```diff
@@ -6,14 +6,26 @@
         "name": "toggleQuotes",
         "type": "function",
         "async": true,
-        "parameters": []
+        "parameters": [
+          {
+            "type": "integer",
+            "name": "windowId",
+            "optional": true
+          }
+        ]
```

Comment:
We pass in the id of the window now and not use the "mostRecentWindow" as before.